app/move/controllers/create/assessment.js

```diff
@@ -2,33 +2,15 @@ const { flatten, values } = require('lodash')
 
 const CreateBaseController = require('./base')
 const fieldHelpers = require('../../../../common/helpers/field')
-const referenceDataService = require('../../../../common/services/reference-data')
-const referenceDataHelpers = require('../../../../common/helpers/reference-data')
 
 class AssessmentController extends CreateBaseController {
   async configure(req, res, next) {
     try {
       const { fields } = req.form.options
 
       await Promise.all(
```

Comment:
Just looking at how this has been refactored to use async/await in the map, does this still need to be a `Promise.all`?

Reply:
`fieldHelpers.mapAssessmentQuestion` doesn't seem to be returning a promise any more so might not be needed.

The only other thing I'm wondering is if this has actually made what was more of an async set of statements into sync ones?

Reply:
> Just looking at how this has been refactored to use async/await in the map, does this still need to be a `Promise.all`?

so the `fieldHelpers.mapAssessmentQuestion` method:
```
async function mapAssessmentQuestion([key, field]) {
 ...
```
 is returning a promise due to the `async`. Therefore the `map` is returning an array of `Promises`. So yup `Promise.all` is very much needed.

> The only other thing I'm wondering is if this has actually made what was more of an async set of statements into sync ones?

Nope as far as I am aware these are `Promises` so all still `async` but will work as `sync` due to the 
```
await Promise.all(
  ...
```